docs/ABOUT.md

```diff
@@ -0,0 +1,17 @@
+José Valim, the creator of Elixir, explains [here](https://vimeo.com/53221562) how he built the language for applications to be:
+1) Distributed
+2) Fault-Tolerant
+3) Soft-Real-Time
+4) Hot-Code-Swapped (can introduce new code without stopping the sever)
+
+[Elixir](http://elixir-lang.org/), initially released in 2012, extends upon the already robust features of Erlang while also being easier for beginners to access, read, test, and write.
```

Comment:
ah okay I was confused at one of the earlier suggestions. I think I overwrote my markdown syntax fix at some point as well. I changed it back to using "." instead of the "("